pyrokinetics/tests/equilibrium/test_equilibrium.py

```diff
@@ -5,29 +5,28 @@
 
 from pyrokinetics.equilibrium import Equilibrium
 from pyrokinetics import Pyro
+from pyrokinetics import template_dir
 import numpy as np
-import pathlib
 import pytest
 
 
 @pytest.fixture(scope="module")
 def geqdsk_equilibrium():
-    template_dir = pathlib.Path(__file__).parent / "../templates"
-    return Equilibrium(eq_type="GEQDSK", eq_file=template_dir / "test.geqdsk")
+    return Equilibrium(eq_type="GEQDSK", eq_file=template_dir.joinpath("test.geqdsk"))
 
 
 @pytest.fixture(scope="module")
 def transp_cdf_equilibrium():
-    template_dir = pathlib.Path(__file__).parent / "../templates"
     return Equilibrium(
-        eq_type="TRANSP", eq_file=template_dir / "transp_eq.cdf", time=0.2
+        eq_type="TRANSP", eq_file=template_dir.joinpath("transp_eq.cdf"), time=0.2
```

Comment:
Just curious, is there a difference between `.joinpath` and `/`?

Reply:
Happy with `joinpath`